src/Kentico.Xperience.Lucene/LuceneSearchModule.cs

```diff
@@ -19,10 +21,8 @@ internal class LuceneSearchModule : Module
 {
     private ILuceneTaskLogger luceneTaskLogger = null!;
     private IAppSettingsService appSettingsService = null!;
-    private IConversionService conversionService = null!;
-
-    [Obsolete("This setting will be replaced in the next major version. Use CMSLuceneSearchDisableIndexing instead.")]
-    private const string APP_SETTINGS_KEY_INDEXING_DISABLED_OLD = "LuceneSearchDisableIndexing";
+    private IConversionService conversionService = null!;
+
     private const string APP_SETTINGS_KEY_INDEXING_DISABLED = "CMSLuceneSearchDisableIndexing";
```

Comment:
Do we want to take the opportunity to create a section for settings here?

```json
{
   "CMSLuceneSearch": {
      "DisableIndexing": true
   }
}
```

I'm ok leaving it as `CMSLuceneSearchDisableIndexing`, but the above approach lets us keep future settings (if we have any) more organized.

Reply:
I agree that it keeps the settings better organized, but it also adds unnecessary complexity (since we only have one setting so far). Also this would be an unnecessary breaking change so I'd rather wait until we have a real reason to organize more than one setting.